Dockerfile

```diff
@@ -55,8 +61,8 @@ RUN luarocks install --deps-mode=none --tree /usr/local https://luarocks.org/man
 RUN luarocks install --deps-mode=none --tree /usr/local https://luarocks.org/manifests/hamish/lua-resty-iputils-0.3.0-1.src.rock
 RUN luarocks install --deps-mode=none --tree /usr/local https://luarocks.org/manifests/golgote/net-url-0.9-1.src.rock
 
-RUN yum -y remove libyaml-devel m4 openssl-devel git gcc luarocks && \
-    rm -rf /var/cache/yum && yum clean all -y && \
+RUN microdnf -y remove yum-utils libyaml-devel m4 openssl-devel perl-Git-2.31.1-2.el8.noarch git annobin-10.67-3.el8.x86_64 gcc-plugin-annobin-8.5.0-16.el8_7.x86_64 gcc luarocks && \
```

Comment:
gonna wildcard the packages we specify with version hard coded in them. building locally just now I got an error that perl-Git-2.31.1-2... doesnt exist and its because perl-Git-2.31.1-3... is the version that was installed. So I can wildcard like this '`perl-Git-*'

- [ ] wildcard perl-Git, annobin, and gcc

Reply:
Done